recipe/conda_build_config.yaml

```diff
@@ -1,9 +1,11 @@
+c_compiler:            # [win]
+- vs2019               # [win]
+cxx_compiler:          # [win]
+- vs2019               # [win]
```

Comment:
Why is this needed? This would make downstream packages linking statically to do the same

Reply:
It's the (new) minimum to build LLVM 14: https://reviews.llvm.org/D114639

Reply:
> It's the (new) minimum to build LLVM 14: https://reviews.llvm.org/D114639

@isuruf, any comment on this how you'd like to proceed with this?

Unsurprisingly I'm not ecstatic about trying to stem the tide here - I think the sane choice is to let vs2017 fade away. On top of that, it's not like clang is pervasively used on windows in our recipes, and we could stay on LLVM <14 for some time yet where necessary.

Reply:
That's fine. There are other issues with using VS2019. See https://github.com/MacPython/numpy-wheels/issues/145